test/benchmarks/benchmark_helper.rb

```diff
@@ -0,0 +1,11 @@
+module BenchmarkHelper
+  def iterate(&block)
+    start = Time.now
+    count = 0
+    while Time.now - start <= 1.second do
+      block.call
```

Comment:
As I learned just now: We should use `yield` instead of `block.call` since `block.call` is almost 2x slower

source: https://stackoverflow.com/questions/1410160/ruby-proccall-vs-yield

Reply:
nice catch. I added an issue for it here https://github.com/procore/painted-rabbit/issues/15